examples/03_blade/analysis_options_aero.yaml

```diff
@@ -21,6 +28,11 @@ constraints:
         stall:
             flag: True     # Constraint on minimum stall margin
             margin: 0.087 # Value of minimum stall margin in [rad]
+        chord:
```

Comment:
A little confused why chord would have its own constraint.  This would be easier to implement as a design variable bound, right?  Or maybe given your multiplier DV approach to the chord it needs to have a max value constraint?

Reply:
yep the bounds are multiplicative (+30% wrt the initial value, -50%, etc), whereas the absolute value can be constrained here. I haven't changed this line actually, only changed the capitalized letter

Reply:
Okay, sounds good.